samples/certs/generate-workload.sh

```diff
@@ -15,17 +15,26 @@
 #   limitations under the License.
 
 name=${1:-foo}
-san="spiffe://trust-domain-$name/ns/$name/sa/$name"
+ns=${2:-$name}
+sa=${3:-$name}
+dir=${4:-"."}
+tmp=${5:-""}
+san="spiffe://trust-domain-$name/ns/$ns/sa/$sa"
 
-openssl genrsa -out "workload-$name-key.pem" 2048
+if [ ! -d "$dir/$tmp" ]; then
+  mkdir "$dir/$tmp"
+fi
 
-cat > workload.cfg <<EOF
+openssl genrsa -out "$dir/$tmp/workload-$name-key.pem" 2048
+
+cat > "$dir"/workload.cfg <<EOF
```

Comment:
this should be under TMP_DIR. to be consistent with others.

Reply:
The tmp dir only contains the temp certs, on purposely keep other files still in current dir which not impacts original script